prefect_saturn/messages.py

```diff
@@ -11,6 +11,10 @@ class Errors:
     where multiple methods use the same error message.
     """
 
+    BASE_URL_NO_SLASH = (
+        "Because this client uses Webhook storage, "
+        "environment variable BASE_URL must end with a forward slash."
```

Comment:
```suggestion
        "environment variable BASE_URL must not end with a forward slash."
```
Right?

Reply:
UGH yeah I'm confusing myself haha. This suggestion is right.